WORKSPACE

```diff
@@ -54,6 +54,11 @@ load("@rules_pkg//:deps.bzl", "rules_pkg_dependencies")
 
 rules_pkg_dependencies()
 
+# For determining available Bazel features
+load("@bazel_features//:deps.bzl", "bazel_features_deps")
+
+bazel_features_deps()
```

Comment:
This should be called in `oci_register_toolchains` macro otherwise it's a breaking change.

Reply:
@thesayyn could you have another look?